packages/web/components/Modal/RegisterModal/index.js

```diff
@@ -38,6 +40,40 @@ const RegisterModal = ({ closeModal }) => {
 		}
 	};
 
+	const [acceptedTerms, setAcceptedTerms] = useState({
+		terms_conditions: false,
+		data_conditions: false,
+		platform_conditions: false,
+		services_conditions: false,
+		channel_conditions: false,
+		link_conditions: false,
+		process_conditions: false,
+	});
+
+	// eslint-disable-next-line consistent-return
+	const handleAcceptedTerms = (type) => {
+		const types = Object.keys(acceptedTerms);
+
+		if (!type || !types.includes(type)) return;
+
+		setAcceptedTerms({
+			...acceptedTerms,
+			[type]: !acceptedTerms[type],
+		});
+	};
```

Comment:
```suggestion
	const handleAcceptedTerms = (type) => {
		const types = Object.keys(acceptedTerms);

		if (!type || !types.includes(type)) return;

		return setAcceptedTerms({
			...acceptedTerms,
			[type]: !acceptedTerms[type],
		});
	};
```

Don't need to disable this rule, just return

Reply:
@luizeboli Mas nesse caso essa arrow function não espera nenhum valor de retorno

Reply:
Essa regra só estourou porque você fez um early return , mas ok 👍